Assets/MRTK/Providers/XRSDK/Profiles/DefaultXRSDKInputSystemProfile.asset

```diff
@@ -21,6 +21,13 @@ MonoBehaviour:
     priority: 0
     runtimePlatform: 9
     deviceManagerProfile: {fileID: 0}
+  - componentType:
+      reference: Microsoft.MixedReality.Toolkit.XRSDK.WindowsMixedReality.OculusXRSDKDeviceManager,
+        Assembly-CSharp
```

Comment:
Looks like this is currently in the default assembly. Don't forget to [make an assembly definition](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Input/CreateDataProvider.html#create-assembly-definitions)!

Reply:
With the new assembly, you'll need to double check the profile and likely reserialize